common/lib/xmodule/xmodule/video_module/transcripts_utils.py

```diff
@@ -654,16 +654,17 @@ def convert(content, input_format, output_format):
         if input_format == 'srt':
 
             if output_format == 'txt':
-                text = SubRipFile.from_string(content.decode('utf8')).text
+                text = SubRipFile.from_string(content.decode('utf-8')).text
                 return HTMLParser().unescape(text)
 
             elif output_format == 'sjson':
+                content = content.decode('utf-8') if six.PY2 else content
                 try:
                     # With error handling (set to 'ERROR_RAISE'), we will be getting
                     # the exception if something went wrong in parsing the transcript.
                     srt_subs = SubRipFile.from_string(
                         # Skip byte order mark(BOM) character
-                        content.decode('utf-8-sig'),
```

Comment:
I think this changes the behavior of the code.   We want to make sure that it skips the BOM characters if they exist when decoding.